### PR TITLE
Note that we've filled the newsletter editor role

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

**Review by mxsasha — APPROVED** [review]
If you want to keep as is I'm fine with that, but maybe remove the interested section and move the note to the top?

**Comment by ericholscher** [comment]
@mxsasha Took another shot.

File: docs/blog/newsletter-editor.rst [hunk 1]
```diff
@@ -9,6 +9,8 @@ We're looking for a new editor to join the `team </team/#newsletter>`__ that cre
 
 Our outgoing editor (Beth Aitman) has loved being on the team and would highly recommend it. It's a great opportunity to help a wide audience of documentarians, by sharing the knowledge from Slack in a more permanent form; it's also a fun way to practise or experiment with a different style of writing. The format isn't set in stone either - we welcome new ideas for how to make the newsletter even better.
 
+**Note: This position has been filled.** Thanks to everyone who applied. We're still excited to have you join our community though -- so feel free to reach out and we can help you find a way to contribute!
```
Comment (plaindocs) [inline docs/blog/newsletter-editor.rst]:
```suggestion
**Note: This position has been filled.** Thanks to everyone who applied. We're still excited to have you join our community though -- please reach out so we can help you find a way to contribute!
```

**Comment by thatdocslady** [comment]
What about linking to the new and improved Team page with all the teams that people can join?

**Comment by mxsasha** [comment]
> What about linking to the new and improved Team page with all the teams that people can join?

I like that, though this is not a new post, just an edit, so I don't think tons of people will still see this.

**Comment by ericholscher** [comment]
Agreed. This is just a small update. If we want more visibility there's other better places.

**Pull request merged**